components/operators/authorino-operator/operator/overlays/managed-services/patch-channel.yaml

```diff
@@ -1,3 +1,3 @@
 - op: replace
   path: /spec/channel
-  value: 'tech-preview-v1'
+  value: 'stable'
```

Comment:
Let's update the overlays folder name from `managed-services` -> `stable`

Be sure to update the references to the overlay in the operator list as well (I know you did not need to modify the fast channel previously, which you will after this).